### PR TITLE
docs: Minor docs tweaks

### PR DESCRIPTION
**Pull request by sbrugman**
## Checklist

- [X] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [X] Updated the documentation to reflect the code changes
- [X] Added a description of this change in the relevant `RELEASE.md` file
- [X] Added tests to cover my changes

### CODE REVIEW

File: kedro-docker/CONTRIBUTING.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ def count_truthy(elements: List[Any]) -> int:
 We use a branching model that helps us keep track of branches in a logical, consistent way. All branches should have the hyphen-separated convention of: `<type-of-change>/<short-description-of-change>` e.g. `feature/awesome-new-feature`
 
 | Types of changes | Description                                                                 |
-| ---------------- | --------------------------------------------------------------------------- |
+|------------------|-----------------------------------------------------------------------------|
```
Comment (astrojuanlu) [inline kedro-docker/CONTRIBUTING.md:63]: I see no difference in result before/after

File: kedro-telemetry/RELEASE.md [hunk 2]
```diff
@@ -6,21 +6,21 @@
 
 The `kedro-telemetry` plugin asks for consent to collect command run data. However, when we introduced functionality to understand the size of projects by counting the number of datasets, pipelines and nodes, the plugin did not verify consent, leading to the plugin collecting data without user consent. This bug affects users with `kedro-telemetry >= 0.2.2` installed, 22.5% of our user base.
 
-<img width="730" alt="Screenshot 2023-04-04 at 16 17 43" src="https://user-images.githubusercontent.com/43755008/229840293-fea91403-b02d-4221-9167-71d3a6196cc5.png">
+<img width="730" src="https://user-images.githubusercontent.com/43755008/229840293-fea91403-b02d-4221-9167-71d3a6196cc5.png">
```
Comment (astrojuanlu) [inline kedro-telemetry/RELEASE.md:9]: I'd rather not remove the alt text, but it's true the previous one was not very helpful 😄

File: kedro-telemetry/RELEASE.md [hunk 3]
```diff
@@ -6,21 +6,21 @@
...
-|_(Hashed)_ Package name| c7cd944c28cd888904f3efc2345198507...|
-|_(Hashed)_ Project name| a6392d359362dc9827cf8688c9d634520e...|
-|`kedro` project version| 0.18.4|
-|`kedro-telemetry` version| 0.2.3|
-|Python version| 3.8.10 (default, Jun  2 2021, 10:49:15) |
-|Operating system used| darwin |
-| Number of datasets | 7 |
-| Number of pipelines | 2 |
-| Number of nodes | 12 |
+| Description               | What we received                        |
```
Comment (astrojuanlu) [inline kedro-telemetry/RELEASE.md:13]: I see no difference in markup before/after

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Thanks a lot for these small tweaks @sbrugman ! However I wouldn't touch the markup unless the rendering is broken, and there is one commend about the alt text of an image

File: kedro-telemetry/RELEASE.md (hunk 3 above)
Comment (sbrugman) [reply kedro-telemetry/RELEASE.md:13]: The rendering is the same. The spacing makes the table readable in text mode in the editor.

File: kedro-docker/CONTRIBUTING.md (hunk 1 above)
Comment (sbrugman) [reply kedro-docker/CONTRIBUTING.md:63]: The rendering is the same. The spacing makes the table readable in text mode in the editor.

**Review by astrojuanlu — APPROVED** [review]
You know what, LGTM 😄

**Review by merelcht — APPROVED** [review]
Thanks for the contribution @sbrugman 👍

**Pull request merged**